site/sitecontent/terms.md

```diff
@@ -0,0 +1,264 @@
+
+# Terms of Service
+
+*Those terms of service are based on those of WordPress.com with changes from Solderpad.com which both shared them under a Creative Commons Sharealike license. The following terms are also licensed as [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).*
+
+Terms and conditions for using the LibreCores website ("Service"). For the purposes of these terms and conditions the term Service also includes the API and any other data source supplying the website.
+
+## Agreement between the user and LibreCores
+
+LibreCores is a website by Free and Open Source Silicon Foundation C.i.C. ("FOSSi Foundation", "we", "us" or "our" for short), a community-interest company registers in England (No. 09848956). Our registeres office is at 71-75 Shelton Street, London, WC2H 9JQ.
+
+The following terms and conditions govern all use of the Service and all content, services and products available at or through the Service. The Service is owned and operated by FOSSi Foundation. The Service is offered subject to your acceptance without modification of all of the terms and conditions contained herein and all other operating rules, policies and procedures that may be published from time to time on this website by FOSSi Foundation (collectively, the “Agreement”).
+
+Please read this Agreement carefully before accessing or using the Service. By accessing or using any part of the Service, you agree to become bound by the terms and conditions of this agreement. If you do not agree to all the terms and conditions of this agreement, then you may not access the Service or use any services. If these terms and conditions are considered an offer by FOSSi Foundation, acceptance is expressly limited to these terms.
+
+## Account Terms
+
+When using the Service to host projects, you are responsible for maintaining the security of your account and any projects, and you are fully responsible for all activities that occur under the account and any other actions taken in connection with projects.
+
+Your account must not be named in a manner that misleads other users of the Service into thinking that you are another person, organisation or company.
+
+You must not describe or assign keywords to projects in a misleading or unlawful manner, including in a manner intended to trade on the name or reputation of others, and FOSSi Foundation may change or remove any description or keyword that it considers inappropriate or unlawful, or otherwise likely to cause FOSSi Foundation liability. You must immediately notify FOSSi Foundation of any unauthorized uses of your projects, your account or any other breaches of security. FOSSi Foundation will not be liable for any acts or omissions by You, including any damages of any kind incurred as a result of such acts or omissions.
```

Comment:
What if I register with a name "coolxyzguy" and tomorrow someone becomes famous as "coolxyzguy" in places outside librecores ? Does that apply to that case ?

Reply:
I would tend to say it strongly depends. In the case you describe the original user can keep the name. This clause basically protects from people knowingly using misleading names.